blueprints/automation/elegoo_printer/elegoo_printer_progress.yaml

```
@@ -39,52 +31,69 @@ blueprint:
+    first }}
+  remaining_print_time_entity: >-
+    {{ device_entities(elegoo_printer_device) | select('search', '_remaining_print_time') |
+    first }}
   file_name_entity: >-
     {{ device_entities(elegoo_printer_device) | select('search', '_file_name') |
     first }}
+  current_status_entity: >-
+    {{ device_entities(elegoo_printer_device) | select('search', '_current_status') |
+    first }}
+  camera_entity: >-
+    {{ camera_entity_input if camera_entity_input != '' else 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)